hl/src/H5IM.c

```diff
@@ -169,6 +169,11 @@ find_palette(H5_ATTR_UNUSED hid_t loc_id, const char *name, H5_ATTR_UNUSED const
     if (name == NULL)
         return -1;
 
+    /* Shut compiler up */
+    (void)loc_id;
+    (void)ainfo;
+    (void)op_data;
+
```

Comment:
The H5_ATTR_UNUSED didn't do the trick? Maybe it needs to go between the type and the parameter name? If so, funny if the use above doesn't produce an error.

Reply:
In fact it does. I removed the redundancy.